sftpgo/get_admin_api_key.sh

```diff
@@ -15,10 +15,12 @@ fi
 
 source ${SETTINGS_PATH}
 
-if [[ ! $(grep "!sftpgo_admin_api_key!" ${SETTINGS_PATH}) ]]; then
-  echo "SFTPgo admin API key already set in NCA settings"
-  $(dirname $0)/view_api_keys.sh
-  exit 0
+if [[ ${1:-} != '--force' ]]; then
+  if [[ ! $(grep "!sftpgo_admin_api_key!" ${SETTINGS_PATH}) ]]; then
```

Comment:
Doesn't this logic still require changing the value in the settings file back to having `!sftpgo_admin_api_key!` in addition to using a `force` flag? :thinking:

We want `||` between the `force` flag check and grep checking for the default string instead of nested `if`'s to enable `force` to just replace an existing API key value, yes?

Reply:
Nope, that's what that first check is for - if `--force` is present, the second check never occurs.

Reply:
Oh, I'm a dummy. I kept reading that in my mind as checking `==` not `!=` :facepalm: